constellations/orion/src/repos/migrations/000021_add_price_per_session_at_classes.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE classes
+    ADD COLUMN price_per_session INT UNSIGNED;
```

Comment:
nice! looks good. By default, adding a column with any type can have a NULL value.
You can NOT allow null values by adding `NOT NULL` and/or `default 0`.